apps/test/unit/templates/curriculumCatalog/curriculumCatalogCardTest.jsx

```diff
@@ -63,6 +63,7 @@ describe('CurriculumCatalogCard', () => {
       scriptId: 1,
       isSignedOut: true,
       onQuickViewClick: () => {},
+      isTeacher: false,
```

Comment:
Since the curriculum catalog is built more for teachers, I think we want to make most of the tests run under `isTeacher: true` and then specifically set it to false below for a separate test that checks a student view only sees the course button not the assign button. Even though it only affects this one change now, we may make future teacher vs. students changes.